lib/jekyll/tags/link.rb

```diff
@@ -16,11 +16,15 @@ def render(context)
           return document.url if document.relative_path == @relative_path
         end
 
-        raise ArgumentError, "Could not find document '#{@relative_path}' in tag '#{TagName}'.\n\n" \
-          "Make sure the document exists and the path is correct."
+        raise ArgumentError, <<eos
+Could not find document '#{@relative_path}' in tag '#{TAG_NAME}'.
+
+
```

Comment:
Do we want *two* blank lines here, or just one?

It looks like the first `\n` terminated the first line, and the second `\n` inserted one blank line, but I might be reading it wrong.